default_warehouse_from_sale_team/i18n/es.po

```diff
@@ -51,6 +51,11 @@ msgstr ""
 "Permite al usuario poder ver todos los tippos de operación, sin importar sus"
 " equipos de ventas"
```

Comment:
Shouldn't be "Equipos de venta"?

Reply:
This change is not mine 🙈

Reply:
Sorry, I meant your change, you also used "equipos de ventas". But I just notices this is also the native translation, so resolving.